common/lib/xmodule/xmodule/js/src/word_cloud/word_cloud_main.js

```diff
@@ -154,136 +154,176 @@
+         * along with all of the top words.
+         *
+         * @param {array} words An array of objects. Each object must have two properties. One property
+         * is 'text' (the actual word), and the other property is 'size' which represents the number that the
+         * word was enetered by the students.
+         *
+         * @param {array} bounds An array of two objects. First object is the top-left coordinates of the bounding
+         * box where all of the words fir, second object is the bottom-right coordinates of the bounding box. Each
+         * coordinate object contains two properties: 'x', and 'y'.
+         */
         WordCloudMain.prototype.drawWordCloud = function(response, words, bounds) {
-            // Color words in different colors.
+                // Color words in different colors.
             var fill = d3.scale.category20(),
 
-            // Will be populated by words the user enetered.
+                // Will be populated by words the user enetered.
                 studentWordsKeys = [],
 
-            // Comma separated string of user enetered words.
+                // Comma separated string of user enetered words.
                 studentWordsStr,
 
-            // By default we do not scale.
+                // By default we do not scale.
                 scale = 1,
 
-            // Caсhing of DOM element
+                // Caсhing of DOM element
                 cloudSectionEl = this.wordCloudEl.find('.result_cloud_section'),
 
-            // Needed for caсhing of d3 group elements
+                // Needed for caсhing of d3 group elements
                 groupEl;
 
-        // If bounding rectangle is given, scale based on the bounding box of all the words.
+            // If bounding rectangle is given, scale based on the bounding box of all the words.
             if (bounds) {
                 scale = 0.5 * Math.min(
-                this.width / Math.abs(bounds[1].x - this.width / 2),
-                this.width / Math.abs(bounds[0].x - this.width / 2),
-                this.height / Math.abs(bounds[1].y - this.height / 2),
-                this.height / Math.abs(bounds[0].y - this.height / 2)
-            );
+                    this.width / Math.abs(bounds[1].x - this.width / 2),
+                    this.width / Math.abs(bounds[0].x - this.width / 2),
+                    this.height / Math.abs(bounds[1].y - this.height / 2),
+                    this.height / Math.abs(bounds[0].y - this.height / 2)
+                );
             }
 
             $.each(response.student_words, function(word, stat) {
                 var percent = (response.display_student_percents) ? ' ' + (Math.round(100 * (stat / response.total_count))) + '%' : '';
 
-                studentWordsKeys.push('<strong>' + word + '</strong>' + percent);
+                studentWordsKeys.push(HtmlUtils.interpolateHtml(
```

Comment:
"Extreme nit" meant that what you have is correct, but it could be written more succinctly (and perhaps readably) in another way.